gene_finder.py

```diff
@@ -22,145 +22,238 @@ def shuffle_string(s):
 
 def get_complement(nucleotide):
     """ Returns the complementary nucleotide
-
         nucleotide: a nucleotide (A, C, G, or T) represented as a string
         returns: the complementary nucleotide
     >>> get_complement('A')
     'T'
     >>> get_complement('C')
     'G'
+    >>> get_complement('T')
+    'A'
+    >>> get_complement('G')
+    'C'
     """
     # TODO: implement this
-    pass
-
+    if nucleotide == 'A':
+        return 'T'
+    elif nucleotide == 'T':
+        return 'A'
+    elif nucleotide == 'C':
+        return 'G'
+    else:
+        return 'C'
```

Comment:
Stylistic comment: Instead of using a bunch of `if` and `else` statements and now that you've learned what dictionaries are, you could do something like this as well :
`nucleotide_complements' = {'A':'T', 'T':'C', 'C':'G', 'G':'C'}`
`return nucleotide_complements[nucleotide]`